### PR TITLE
Rename subsystems into components

### PR DESCRIPTION
**Pull request by rpelisse**
@sabre1041 give it 20' to see if it passes CI and then please review and merge!

### CODE REVIEW

File: plugins/action/jcliff.py [hunk 1]
```diff
@@ -70,8 +70,12 @@ def _lookup_subsys_template(self, subsys_name):
 
     def _build_and_deploy_jcliff_rule_files(self, tmp_remote_src):
         display.vvvv(u"Build and deploy jcliff rule tmpfile: %s" % tmp_remote_src)
-        display.vvvv(u"Subsystems: %s" % self._task.args['subsystems'])
-        subsystems = self._task.args['subsystems']
+
+        if 'subsystems' in self._task.args:
```
Comment (sabre1041) [inline plugins/action/jcliff.py:74]: By specifying the alias, both options should map to the `components` variable so we should not need the conditional check

File: plugins/action/jcliff.py [hunk 2]
```diff
@@ -70,8 +70,12 @@ def _lookup_subsys_template(self, subsys_name):
...
+            subsystems = self._task.args['subsystems']
+        if 'components' in self._task.args:
+            subsystems = self._task.args['components']
+
         if subsystems is not None:
```
Comment (sabre1041) [inline plugins/action/jcliff.py:79]: Should change variable to components

**Review by sabre1041 — CHANGES_REQUESTED** [review]
A couple requests

File: plugins/action/jcliff.py (hunk 1 above)
Comment (rpelisse) [reply plugins/action/jcliff.py:74]: I've tried, did not work... Maybe you can try and let me know if it works for you?

File: plugins/action/jcliff.py (hunk 2 above)
Comment (rpelisse) [reply plugins/action/jcliff.py:79]: Yes, and this is why my previous change already. But I would rather do that in a separate commit/PR.

File: plugins/action/jcliff.py (hunk 1 above)
Comment (sabre1041) [reply plugins/action/jcliff.py:74]: it works for me without issue

Comment (rpelisse) [reply plugins/action/jcliff.py]: CI seems to disagree with your statement. The error is that there is no rule file generated for jcliff (which means the if 'subsystems' in self._task.args: was false and the all rule generation was skipped).

Can you double check ? And maybe update the PR ?

Comment (sabre1041) [reply plugins/action/jcliff.py]: @rpelisse heres a superset of this PR -> https://github.com/wildfly-extras/ansible_collections_jcliff/pull/46

**Pull request closed without merging**